code/src/components/inputTypes/TextField.css

```diff
@@ -0,0 +1,35 @@
+input[type="text"] {
+  border: none;
+  width: 300px;
+  margin: 20px auto 10px auto;
+  background-color: #cecece;
+  border-bottom: 2px solid var(--prim);
+  font-size: 24px;
+}
+
+::placeholder {
```

Comment:
Do you mean default border on the text-field? Because there is none :)

![grafik](https://user-images.githubusercontent.com/77005046/139573365-fc315cde-85bc-4088-b41c-fc5fc0bc0289.png)

Reply:
I mean the blue outline that shows when the user clicks on the input field. I took it away by writing this (but as I said it is not necessary, I think it look good either way :)):
`input:focus,
select:focus {
  outline: none;
`

Reply:
Ah now I understand.
Thanks for the tip! :)